org.gnomeradio.Radio.json

```
@@ -0,0 +1,52 @@
+{
+  "app-id" : "org.gnomeradio.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "45",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "rename-icon" : "hicolor/scalable/apps/gnome-radio.svg",
```

From the documentation:
```
       rename-icon (string)
           Any icon with this name will be renamed to a name based on id during the cleanup phase. Note that this
           is the icon name, not the full filenames, so it should not include a filename extension.
```

```suggestion
  "rename-icon" : "gnome-radio",
```

---

From a previous build log
```
Error: icon scalable/apps/gnome-radio.svg not found below /srv/buildbot/worker/build-x86_64-1/build/.flatpak-builder/rofiles/rofiles-LS0l1B/files/share/icons
```